### PR TITLE
Update architectural-principles.md

### PR DESCRIPTION
**Pull request by kv-prajapati**
Use the word 'class' in place of 'module'

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/architecture/modern-web-apps-azure/architectural-principles.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ In classes, encapsulation is achieved by limiting outside access to the class's
 
 ### Dependency inversion
 
-The direction of dependency within the application should be in the direction of abstraction, not implementation details. Most applications are written such that compile-time dependency flows in the direction of runtime execution, producing a direct dependency graph. That is, if module A calls a function in module B, which calls a function in module C, then at compile time A will depend on B, which will depend on C, as shown in Figure 4-1.
+The direction of dependency within the application should be in the direction of abstraction, not implementation details. Most applications are written such that compile-time dependency flows in the direction of runtime execution, producing a direct dependency graph. That is, if class A calls a method of class B and class B calls a method of class Ce C, then at compile time class A will depend on classB, and class B will depend on C, as shown in Figure 4-1.
```
Comment (ardalis) [inline docs/architecture/modern-web-apps-azure/architectural-principles.md:31]: remove `Ce` and change `classB` to `class B` please.

Comment (kv-prajapati) [reply docs/architecture/modern-web-apps-azure/architectural-principles.md]: Thanks! Done

File: docs/architecture/modern-web-apps-azure/architectural-principles.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ In classes, encapsulation is achieved by limiting outside access to the class's
 
 ### Dependency inversion
 
-The direction of dependency within the application should be in the direction of abstraction, not implementation details. Most applications are written such that compile-time dependency flows in the direction of runtime execution, producing a direct dependency graph. That is, if module A calls a function in module B, which calls a function in module C, then at compile time A will depend on B, which will depend on C, as shown in Figure 4-1.
+The direction of dependency within the application should be in the direction of abstraction, not implementation details. Most applications are written such that compile-time dependency flows in the direction of runtime execution, producing a direct dependency graph. That is, if class A calls a method of class B and class B calls a method of class C, then at compile time class A will depend on class B, and the class B will depend on class C, as shown in Figure 4-1.
```
Comment (ardalis) [inline docs/architecture/modern-web-apps-azure/architectural-principles.md:31]:
```suggestion
The direction of dependency within the application should be in the direction of abstraction, not implementation details. Most applications are written such that compile-time dependency flows in the direction of runtime execution, producing a direct dependency graph. That is, if class A calls a method of class B and class B calls a method of class C, then at compile time class A will depend on class B, and class B will depend on class C, as shown in Figure 4-1.
```

**Comment by ardalis** [comment]
You closed this without merging it?

**Comment by kv-prajapati** [comment]
> You closed this without merging it?

Sorry about that. Reopend :)

**Comment by ardalis** [comment]
Should be good to merge, @nishanil

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**